core/data.go

```diff
@@ -157,10 +157,10 @@ type BlobMessage struct {
 	Bundles    Bundles
 }
 
-func (b Bundle) Size() int64 {
-	size := int64(0)
+func (b Bundle) Size() uint64 {
+	size := uint64(0)
 	for _, chunk := range b {
-		size += int64(chunk.Size())
+		size += chunk.Size()
```

Comment:
Just for your information, referring to yesterdays discussion, all chunk will now contains 32bytes. Nothing wrong here